src/Transforms/Repeater.php

```diff
@@ -0,0 +1,62 @@
+<?php namespace Leean\Acf\Transforms;
+
+/**
+ * Class Repeater.
+ *
+ * @package Leean\Acf\Transforms
+ */
+class Repeater
+{
+	/**
+	 * Apply the transforms
+	 *
+	 * @param array $field The field.
+	 * @return array
+	 */
+	public static function apply( $field ) {
```

Comment:
This doesn't make sense afaik, because it's a static method. Static methods can't be implemented in a child class, in fact I don't think you can even declare an abstract static method.